components/organisms/o-payment.vue

```diff
@@ -116,6 +116,10 @@
         class="form__element"
         name="phone"
         :label="$t('Phone Number')"
+        :required="true"
```

Comment:
Instead of that, you could just use: `required` and it would be shorthand to `:required="true"`

Reply:
Yes, I fixed this.